src/services/bip39-service/bip39-service.spec.ts

```diff
@@ -182,6 +184,56 @@ describe("BIP39Service", () => {
             }
         );
     });
+
+    it("should return a string array from a string and when already retrieved return it locally", (done) => {
+        spyOn(httpClient, "get").and.returnValue(Observable.of("word1, word2, word3"));
+        activeWordListSpy.and.callThrough();
+        (<any>service).getActiveWordList().then(data => {
+            expect(data).toEqual(['word1, word2, word3']);
+
+            (<any>service).activeWordList = data;
+
+            (<any>service).getActiveWordList().then(data => {
+                expect(data).toEqual(['word1, word2, word3']);
+                done();
+            });
+        });
+    });
+
+    it("should throw an exception because the entropy size in bits has to be divisible by 32", (done) => {
+        service.generate(15).then(data => {
+            // Should throw exception
+            expect(false).toBeTruthy();
+            done();
+        }).catch(data => {
+            expect(true).toBeTruthy();
+            done();
+        });
+    });
+
+    it("should return a specific uint8array", () => {
+        spyOn(forge.random, "getBytesSync").and.returnValue("a");
+
+        let intarray = new Uint8Array(1);
+        intarray.set([97]);
+
+        expect((<any>service).getRandomBytes(10)).toEqual(intarray);
+    });
+
+    it("should throw because the bytearray is not correct", () => {
+        expect((<any>service).toMnemonic).toThrowError();
```

Comment:
How does this test work? The `toMnemonic` function is not called, or am I reading this wrong?